app/livechat/server/config.js

```diff
@@ -555,4 +545,18 @@ Meteor.startup(function() {
 		i18nLabel: 'How_long_to_wait_to_consider_visitor_abandonment',
 		i18nDescription: 'Time_in_seconds',
 	});
+
+	settings.add('Livechat_call_provider', 'none', {
```

Comment:
Lets call this `Omnichannel_call_provider` instead, since this setting would apply for other channels as well